forem/forem
# [15 min fix] move seeded flows into correct folder

## What type of PR is this? (check all applicable)


## Description

This is just a quick fix - some E2E tests were merged after the change to the folder structures, and just need popped into the right directory.

## Related Tickets & Documents

N/A

## QA Instructions, Screenshots, Recordings

Not much to check here, if the tests continue to pass then we're all good. When they were merged previously they wouldn't have actually been run by CI since they weren't in the correct folder.

### UI accessibility concerns?

N/A

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
- [X] This change does not need to be communicated, and this is why not: just shuffling some files around 🙂
